webapp-django/crashstats/settings/base.py

```diff
@@ -764,11 +696,11 @@ def path(*dirs):
     'secrets': {
         'postgresql': {
             'database_password': config(
-                'DATASERVICE_DATABASE_PASSWORD',
+                'DATABASE_PASSWORD',
                 'aPassword'
             ),
             'database_username': config(
-                'DATASERVICE_DATABASE_USERNAME',
+                'DATABASE_USERNAME',
```

Comment:
Should this be `DATABASE_USER`? I see instances of `DATABASE_USER`, but not `DATABASE_USERNAME` in the socorro and socorro-infra repositories. Would having the key wrong here affect anything?